src/uglify/index.js

```diff
@@ -1,10 +1,9 @@
 import os from 'os';
+import cacache from 'cacache';
```

Comment:
@michael-ciniawsky i think rename can be misleading, we can use `cache` variable for other purpose. Also i think it is bad practice, but if someone from the reviewers considers renaming a good practice, say it here and I do it :smile: